circuit-test/src/main/kotlin/com/slack/circuit/test/FakeNavigator.kt

```diff
@@ -35,6 +36,11 @@ public class FakeNavigator : Navigator {
     return null
   }
 
+  override fun resetRoot(newRoot: Screen): List<Screen> {
+    newRoots.add(newRoot)
+    return emptyList()
```

Comment:
Could we pop off all the screens from the `navigatedScreens` turbine and return them to make this a bit more functional?

Reply:
I don't see why not. Done